src/Components/Movies/Movies.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+// B O O T S T R A P P I N ~
+import Container from 'react-bootstrap/Container';
+// C S S
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './Movies.css';
+
+class Movies extends React.Component {
+  render() {
+    if (this.props.movies.length === 0) {
+      return ('');
+    }
+    return (
+      <>
+        <Container>
+          <h1>What's been filmed here?</h1>
+          {
+            this.props.movies.map((movie, index) => (
+              <div key={index} >
+                <img src={movie.image_url} alt={movie.title} />
```

Comment:
Overall, I like that your code is mostly readable. Did you get the movie image to work? I've tried a few times, but think the problem is which type of bootstrap styling I apply. We'll see if I can get to it!

Reply:
I did get the image to appear. There's a bootstrap element rendering that image to my App. Happy to lend a hand if you need another set of eyes on your lab.